parrots.json

```diff
@@ -561,5 +561,9 @@
     {
       "hd": "hd/serbiaparrot.gif",
       "name": "Serbian Parrot"
-    }
+    },
+    {
+      "hd": "hd/laptop_parrot.gif",
+      "name": "Laptop Parrot"
+    },
```

Comment:
Oops! Extra comma needs to be trimmed off!

Reply:
My bad, just fixed it :)